zero-loads-app/src/loads/control/control.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from asyncio import Future
+from contextlib import asynccontextmanager
+from typing import Coroutine
+
+from aiomqtt import Client
+from aiomqtt import Client as MqttClient
+from aiomqtt import Message as MqttMessage
+
+from loads.config import Settings
+
+from .message import Conditions, Message, SensorInput
+
+logger = logging.getLogger("control")
+
+
+class LoadsControl:
```

Comment:
This can just be `Control`. The "loads" context is already provided by this being in the `zero-loads-app`. Other classes aren't named `Loads...` either